### PR TITLE
SDK-3328

### PR DESCRIPTION
**Pull request by wiperz1789**
Overriding Android toolchain cmake options.
Fixed: generation of metrics overlaid video file.
Using FPS from input video unless specified by --sfps.

### CODE REVIEW

File: vision/frame-detector-video-demo/VideoDemo.cpp [hunk 1]
```diff
@@ -250,24 +250,25 @@ int main(int argsc, char** argsv) {
             return 1;
         }
 
+        //Get resolution and fps from input video
+        int sniffed_fps, frameHeight, frameWidth;
+        VideoReader::SniffResolution(program_options.input_video_path, frameHeight, frameWidth, sniffed_fps);
+        if (program_options.sampling_frame_rate == 0) {
+            // If user did not specify --output_fps (i.e. // default of -1), used the sniffed_fps
```
Comment (adennie) [inline vision/frame-detector-video-demo/VideoDemo.cpp:257]: should that say "if user did not specify --spfps (i.e. default of 0)"?

File: vision/CMakeLists.txt [hunk 2]
```diff
@@ -18,6 +18,14 @@ SETUP_INSTALL_DIRS()
...
 add_compile_options(-Wno-deprecated -Wno-deprecated-declarations)
 
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11 -pthread ")
+#add custom cxx flags for android
+if (ANDROID)
+    set(CMAKE_CXX_FLAGS "-ftemplate-depth-750 -fvisibility-inlines-hidden -std=c++11 -pthread -O2 -fPIC -fPIE -flto \
+                        -DANDROID -fdata-sections -ffunction-sections -funwind-tables -fstack-protector-strong \
+                        -no-canonical-prefixes -D_FORTIFY_SOURCE=2 -Wformat -Werror=format-security -DNDEBUG -march=armv8-a")
+endif()
+
```
Comment (adennie) [inline vision/CMakeLists.txt:28]: this "if (ANDROID)" part goes away, right?

Comment (wiperz1789) [reply vision/CMakeLists.txt:28]: yes

File: vision/frame-detector-video-demo/VideoDemo.cpp (hunk 1 above)
Comment (wiperz1789) [reply vision/frame-detector-video-demo/VideoDemo.cpp]: yes, fixed it

**Pull request merged**